rdkit/Chem/UnitTestMol3D.py

```diff
@@ -358,5 +358,28 @@ def testIssue2890(self):
 
     assert len(list(AllChem.EnumerateStereoisomers(mol))) == 2
```

Comment:
I'm assuming this is the check for clearing the CIPCodes.

Reply:
this line is just something that's been there for a long time. It really shouldn't be using an `assert`, but it's old